demos/github-demo/src/isograph-components/__isograph/Query/RepositoryPage/entrypoint.ts

```diff
@@ -25,6 +25,12 @@ const queryText = 'query RepositoryPage ($repositoryName: String!, $repositoryOw
           author {\
             __typename,\
```

Comment:
kind of cool that this has the correct behavior (selecting `id` only if `author` is a `User`)